### DISCUSSION TITLE
Same emoji for Until and Version

### DISCUSSION THREAD

**Discussion by YuriDevAT** [jolution/todo-nukem#13] [Q&A] [answered]
Going through the list of **Optional Meta Blocks** I noticed that for Until and Version the same emoji is used. Is this on purpose? Wouldn't that be confusing or was this discussed within the team to make sense in this context?

**Comment by juliankasimir** [comment, marked as answer]
Thanks for pointing that out. It wasn't intentional to use the same emoji for both "Until" and "Version". I agree that this could be confusing.

We changed it to the following, to make it more clear.

- 📅 (calendar) for "Until" to emphasize the deadline aspect.
- 🔖 (bookmark) for "Version" to signify a specific version tag.

We are also happy to receive alternative suggestions.

https://github.com/jolution/todo-nukem/blob/main/README.md#optional-meta-blocks

**Reply by YuriDevAT** [reply]
Seems like a good fit.
